module.xml

```diff
@@ -3,7 +3,7 @@
 <Document name="ZPM.ZPM"><Module>
   <Name>ZPM</Name>
   <Version>0.10.0-SNAPSHOT</Version>
-  <ExternalName>Package Management System</ExternalName>
+  <ExternalName>IPM</ExternalName>
```

Comment:
Nit: Perhaps could have it be InterSystems Package Manager?

Reply:
I want to keep it short because of the additional significance we're giving the internal name (specifically, allowing interchangeable use in commands).